### ISSUE TITLE
Can't eat batteries (not an option) when CBM Battery System is installed

### ISSUE THREAD

**Issue by hmstanley** [CleverRaven/Cataclysm-DDA#19191]
if I'm doing it wrong.. but it reads  "E" batteries to get power.. can't eat batteries in the latest build

mods:
[
    "ew_pack",
    "makeshift",
    "More_Survival_Tools",
    "Medieval_Stuff",
    "more_locations",
    "Experimental-Z-Level",
    "boats",
    "deoxymod",
    "blazemod",
    "Tanks",
    "StatsThroughSkillsFour"
]

**Comment by Chezzo** [comment]
I can confirm this happens on vanilla.

**Comment by codemime** [comment]
After #18802 batteries are no longer considered as ammo by `item::ammo_type()` (along with any other ammo). Absence of [these lines](https://github.com/CleverRaven/Cataclysm-DDA/pull/18802/files#diff-f7ca61cf749eba039da961e8a7be1630L4162) changes the behavior. Adding them back would help, but something else may break then. Pinging @mugling.

**Comment by mugling** [comment]
I thought I had checked for that but obviously not

**Comment by illi-kun** [comment]
Duplicate of #18908, so I'll close this but assign ```0.D``` milestone to the referenced issue.
